main.go

```diff
@@ -58,3 +59,22 @@ func postTasks(c *gin.Context) {
 	tasks = append(tasks, newTask)
 	c.IndentedJSON(http.StatusCreated, newTask)
 }
+
+func putTasks(c *gin.Context) {
```

Comment:
This seems good, but I'm reading some Go docs rn to see if this is correct. I'll approve once I know enough